test/libsolidity/syntaxTests/errors/basic_usage.sol

```diff
@@ -0,0 +1,9 @@
+error E();
+function f() pure {
+    revert(E());
```

Comment:
Why does this looks like executing E?

Reply:
Because it does execute E? Do you want a keyword like for events?

Reply:
In what way does it "execute E"?